src/classes/TDTM_Config.cls

```diff
@@ -37,52 +37,52 @@
     private static Integer rowCount;
-    
+
     /********************************************************************************************************
-    * @description Returns the TDTM configuration. If no configuration is currently set, it will return 
-    * the default configuration without setting it. The configuration returned has not yet been inserted in the 
-    * database. It can be manipulated, if necessary, then set. 
-    * If no configuration exists, the application will insert the defaults when any of the triggers run. 
-    * @return List<TDTM_Global_API.TdtmToken> List of trigger handler tokens with the default values. 
+    * @description Returns the TDTM configuration. If no configuration is currently set, it will return
+    * the default configuration without setting it. The configuration returned has not yet been inserted in the
+    * database. It can be manipulated, if necessary, then set.
+    * If no configuration exists, the application will insert the defaults when any of the triggers run.
+    * @return List<TDTM_Global_API.TdtmToken> List of trigger handler tokens with the default values.
     *********************************************************************************************************/
     public static List<TDTM_Global_API.TdtmToken> getTdtmConfig() {
         if(tdtmTokensCache != null) {
             return tdtmTokensCache;
         }
         List<TDTM_Global_API.TdtmToken> tokens = new List<TDTM_Global_API.TdtmToken>();
-        List<Trigger_Handler__c> tdtmConfig = [select Class__c, Object__c, Trigger_Action__c, Load_Order__c, Active__c, 
+        List<Trigger_Handler__c> tdtmConfig = [select Class__c, Object__c, Trigger_Action__c, Load_Order__c, Active__c,
                             Asynchronous__c, Filter_Field__c, Filter_Value__c, User_Managed__c from Trigger_Handler__c];
-                         
+
         // Getting the default configuration only if there is no data in the Trigger Handler object. Otherwise
-        // we would delete customizations and Trigger Handlers entries that aren't in the default configuration. 
+        // we would delete customizations and Trigger Handlers entries that aren't in the default configuration.
         if(tdtmConfig.size() == 0) {
 	        tdtmConfig = TDTM_DefaultConfig.getDefaultRecords();
         }
-        
+
         for(Trigger_Handler__c handler : tdtmConfig) {
-            tokens.add(new TDTM_Global_API.TdtmToken(handler.Class__c, handler.Object__c, handler.Trigger_Action__c, 
-                    Integer.valueOf(handler.Load_Order__c), handler.Active__c, handler.Asynchronous__c, 
-                    handler.Filter_Field__c, handler.Filter_Value__c, handler.User_Managed__c));
+            tokens.add(new TDTM_Global_API.TdtmToken(handler.Class__c, handler.Object__c, handler.Trigger_Action__c,
+                    Integer.valueOf(handler.Load_Order__c), handler.Active__c, handler.Asynchronous__c,
```

Comment:
Agreed. I made sure the unit tests include a float and the validation checks the value with decimal places is preserved.